packages/components/src/avatar/src/AvatarGroup.tsx

```diff
@@ -43,8 +43,8 @@ function RemainingAvatars({ avatars, size, ...rest }: RemainingAvatarsProps) {
             </Box>
             <Tooltip>
                 <ul className="o-ui-avatar-group-remainings-list">
-                    {avatars.filter((x: ReactElement) => !isNil(x?.props?.name)).map((x: ReactElement) => {
-                        const name = x?.props?.name;
+                    {avatars.filter((x: ReactElement) => !isNil(x?.props?.name) || !isNil(x?.props["aria-label"])).map((x: ReactElement) => {
```

Comment:
the name used to check for null props (x?.props?) so i would be just as defensive on your check. 

!isNil(x?.props?["aria-label"]

Reply:
all 3 places.

Reply:
I'll try it, but if I remember right, it doesn't like the ? there. If it complains, not sure what else to do... It didn't complain how I had it.

Reply:
It shouldn't, let me know if you have an issue